third_party/move/move-core/types/src/account_address.rs

```diff
@@ -15,13 +15,17 @@ use std::{convert::TryFrom, fmt, str::FromStr};
 pub struct AccountAddress([u8; AccountAddress::LENGTH]);
 
 impl AccountAddress {
+    /// Hex address: 0x4
+    pub const FOUR: Self = Self::get_hex_address_four();
```

Comment:
Why not ordering constants for addresses like 1,2,3,4?😄

Reply:
Yeah, it is currently order by the alphabet so four is the first one. I will change that